README.md

```diff
@@ -1,70 +1,49 @@
-# Getting Started with Create React App
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+<h1 align="center"> Covid Country Metrics </h1>
 
-## Available Scripts
+> The application fetches Covid Data from the Narrativa API and shows in a country by country display according to the date you are looking for.
 
-In the project directory, you can run:
+## Built With
 
-### `npm start`
+- **Reactjs** and **Redux** and **Javascript**
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+## DEMO
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+[Live Demo Link](https://heroku_link_goes_here.com)
```

Comment:
As required, Kindly deploy your application properly using Heroku, Netlify or GitHub Pages 👍

Reply:
Deployed and link updated